src/API/Reports/Customers/Controller.php

```diff
@@ -166,12 +166,14 @@ public function get_item( $request ) {
 	 * @return WP_REST_Response
 	 */
 	public function prepare_item_for_response( $report, $request ) {
-		$context                      = ! empty( $request['context'] ) ? $request['context'] : 'view';
-		$data                         = $this->add_additional_fields_to_object( $report, $request );
-		$data['date_registered_gmt']  = wc_rest_prepare_date_response( $data['date_registered'] );
-		$data['date_registered']      = wc_rest_prepare_date_response( $data['date_registered'], false );
-		$data['date_last_active_gmt'] = wc_rest_prepare_date_response( $data['date_last_active'] );
-		$data['date_last_active']     = wc_rest_prepare_date_response( $data['date_last_active'], false );
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->add_additional_fields_to_object( $report, $request );
+		// Registered date is UTC.
+		$data['date_registered_gmt'] = wc_rest_prepare_date_response( $data['date_registered'] );
+		$data['date_registered']     = wc_rest_prepare_date_response( $data['date_registered'], false );
+		// Last active date is local time.
```

Comment:
I'm not going to admit to how many times I had to read the code to spot the difference 🤣